app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<p> here will be details about one post</p>
+<div class="container">
+    <h1>Posts Detail</h1>
+    <div class="post-detail">
+        <div class="post-head">
+            <h5>Post <%=@post.id %> <%= @post.title %> <% 'by' %> <%= @user.name %></h5>
+            <ul>
+            <li class="like-form">
+                    <%= form_with model: like, method: :post, url: new_user_like_path, local: true, scope: :new_like do |form| %>
+                        <%= form.submit "Like", class: "btn btn-primary" %>
+                        <%= form.hidden_field :post_id, value: @post.id %>
+                    <% end %>
+                </li>
+                <li>Comments: <%= @post.comments_counter || 0 %> </li>
+                <li>Likes: <%= @post.likes_counter || 0 %> </li>
+            </ul>
+        </div>
+        <div class="post-body">
+            <p><%= @post.text %></p>
```

Comment:
- [ ] As raised earlier, kindly be consistent with the naming of your variables, this have hindered me to confirm your implementation. ✔️ 

## Schema.rb
<img width="344" alt="Untitled" src="https://github.com/AlaaAlsalem/blogaapp/assets/103386183/a747b7b5-77b3-4bed-8d82-802cdd67dbac">

**Please kindly look through your implementation and resolve any similar cases** 🙏